lib/usb/usb_fx07_common.c

```diff
@@ -228,15 +228,22 @@ uint16_t stm32fx07_ep_read_packet(usbd_device *usbd_dev, uint8_t addr,
 	uint32_t extra;
 
```

Comment:
So, is comment about reading seveeral small chunks, what's the impact of that? is that a change from existing?